src/validate_pyproject/plugins/__init__.py

```diff
@@ -47,6 +47,9 @@ def help_text(self) -> str:
             return ""
         return Template(tpl).safe_substitute(tool=self.tool, id=self.id)
 
+    def __repr__(self) -> str:
+        return f"{self.__class__.__name__}({self.tool!r}, {self._load_fn!r})"
```

Comment:
Thank you very much @henryiii.
I am fine the way this is, but I was wondering what you would think of:

```suggestion
        return f"{self.__class__.__name__}({self.tool!r}, {self.id})"
```

Does it help to differentiate between 2 different functions that share the same name?

Reply:
That was basically what I originally wrote (though I used `self._load_fn.__name__`, but yes that's easier). I don't have a strong opinion (it's mostly for debugging anyway), but especially since it's already available as `.id`, that would be fine too.